### PR TITLE
It looks like done.

### PR DESCRIPTION
**Pull request by OlhaHumeniuk**

### CODE REVIEW

File: git/homework.py [hunk 1]
```diff
@@ -78,7 +82,11 @@ def multiple_ints_with_conversion(first_value: Any, second_value: Any) -> int:
             print("Not valid input data")
         >>> "Not valid input data"
     """
-    pass
+    try:
+        return int(first_value) * int(second_value)
+    except TypeError:
+        raise ValueError
+        print('Not valid input data')
```
Comment (klimach) [inline git/homework.py]: This line will never work

File: git/homework.py [hunk 2]
```diff
@@ -127,7 +135,7 @@ def alphabet() -> dict:
         alphabet()
         >>> {"a": 1, "b": 2 ...}
     """
-    pass
+    return dict(OrderedDict((i+1,k) for i, k in enumerate(ascii_lowercase)))
```
Comment (klimach) [inline git/homework.py]: We can make it easier.
Try this: `dict(enumerate(ascii_lowercase, 1))`

File: git/homework.py (hunk 1 above)
Comment (OlhaHumeniuk) [reply git/homework.py]: Just curious.
Why does test pass?

Comment (klimach) [reply git/homework.py]: When an error arises, the code stops working. Until this point, execution will never reach.

Comment (klimach) [reply git/homework.py]: But if you want ValueError to output the custom message of error, just add this to the arguments.
For example: `ValueError('Not valid input data')`

File: git/homework.py (hunk 2 above)
Comment (OlhaHumeniuk) [reply git/homework.py]: image

Comment (OlhaHumeniuk) [reply git/homework.py]: Seems like ok. Both of the changes proposed.

Comment (OlhaHumeniuk) [reply git/homework.py]: I have deleted 1 unused import & change Value Error message code.

File: git/homework.py (hunk 1 above)
Comment (OlhaHumeniuk) [reply git/homework.py]: done

**Comment by klimach** [comment]
Good job 👍

**Pull request merged**